### DISCUSSION TITLE
release number selection

### DISCUSSION THREAD

**Discussion by grubert** [psincraian/pepy#290] [General]
hei.

i made a beta release of a project and wanted to see if it is downloaded
pepy did not show the newest 0.17b1 but 0.16rc1 and 0.16b0.dev0 
but not 0.16 ... is this intentionally ?

very interesting to me is that rc1 is still downloaded although in microcents of the release
and then there are days 0.14rc2 is downloaded, looks strange to me

thanks for your effort (the project i checked was docutils)

**Comment by psincraian** [comment]
Hey, 

take into account that pepy.tech shows the downloads from the previous day so if you released a package today only tomorrow you will get download stats.

Regarding the versions shown, we are using natsort to show the "last" versions. But it has some bugs as these one.

I'm glad that you like the project
